botocore/model.py

```diff
@@ -235,6 +236,13 @@ def operation_model(self, operation_name):
             raise OperationNotFoundError(operation_name)
         return OperationModel(model, self, operation_name)
 
+    def client_name_to_operation_name(self, name):
```

Comment:
@jamesls 
We can move it up to the client. After looking into dangling these presign methods onto the client, I learned that I actually do not need this method at all. I am inclined to remove it completely in that case. Thoughts?